app/models/event.rb

```diff
@@ -11,6 +11,8 @@ class Event < ActiveRecord::Base
   # validates :description, presence: true
   # validates :category_id, presence: true
 
+  scope :upcoming_events, -> { where("time >= ?", Time.zone.now.beginning_of_day) }
```

Comment:
Already tried seperating into two lines but hound didn't like the scope being in two lines